torch/fx/node.py

```diff
@@ -31,6 +31,12 @@
     BaseArgumentTypes
 ]]
 
+_side_effectful_autograd_functions: Set[Callable] = {
```

Comment:
nit: this is named "..autograd_functions" but contains both autograd and autocast API's